inst/qml/PrincipalComponentAnalysis.qml

```diff
@@ -23,79 +23,79 @@ import JASP.Controls 1.0
 
 Form
 {
-	
 
-	CheckBox { name: "incl_GoF"; checked: true; visible: false }
-	CheckBox { name: "incl_fitIndices"; checked: false; visible: false }
-	CheckBox { name: "incl_loadings"; checked: true; visible: false }
-	IntegerField { name: "plotHeightPathDiagram"; defaultValue: 0; visible: false }
-	IntegerField { name: "plotHeightScreePlot"  ; defaultValue: 300; visible: false }
-	IntegerField { name: "plotWidthPathDiagram" ; defaultValue: 480; visible: false }
-	IntegerField { name: "plotWidthScreePlot"   ; defaultValue: 300; visible: false }
+
+	CheckBox { name: "goodnessOfFit"; checked: true; visible: false }
+	CheckBox { name: "fitIndices"; checked: false; visible: false }
+	CheckBox { name: "loadings"; checked: true; visible: false }
+	IntegerField { name: "pathDiagramPlotheigth"; defaultValue: 0; visible: false }
+	IntegerField { name: "screePlotPlotHeigth"  ; defaultValue: 300; visible: false }
+	IntegerField { name: "pathDiagramPlotWidth" ; defaultValue: 480; visible: false }
+	IntegerField { name: "screePlotPlotWidth"   ; defaultValue: 300; visible: false }
```

Comment:
The same as in efa: do we need those options?

Reply:
I have no idea. I dont think they appear in the R code, so I would just delete them